Sources/KeyboardSupport/KeyboardRespondable.swift

```diff
@@ -44,6 +44,12 @@ extension UIViewController {
     }
 }
 
+/// KeyboardScrollable will ask a TextInputView that conforms to this protocol for preferred distance between the field and the keyboard.
+/// It will be used if it is non-nil and greater than the KeyboardScrollable's minimumPaddingAroundInput'.
+@objc public protocol KeyboardPaddingProviding {
+    @objc var inputPadding: UIEdgeInsets { get }
```

Comment:
What's the reasoning behind the `@objc` for this property and its parent protocol?

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/5088e4249dde73bc0b6cce06fa53e76b9997e3ed